pkg/models/property.go

```diff
@@ -12,9 +12,10 @@ import (
 // This is done simply due to lack of solid generics support in golang.
 type AssetPropertyValueQuery struct {
 	BaseQuery
-	AggregateTypes []string `json:"aggregates,omitempty"` // Not used for the history query
-	Quality        string   `json:"quality,omitempty"`
-	Resolution     string   `json:"resolution,omitempty"`
+	AggregateTypes  []string `json:"aggregates,omitempty"` // Not used for the history query
+	Quality         string   `json:"quality,omitempty"`
+	Resolution      string   `json:"resolution,omitempty"`
+	LastObservation bool     `json:"lastObservation,omitempty"`
```

Comment:
```suggestion
	Interpolate string  `json:"interpolate,omitempty"`
```

Given that the property is defined as:
```LINEAR_INTERPOLATION | LOCF_INTERPOLATION```
https://docs.aws.amazon.com/iot-sitewise/latest/APIReference/API_GetInterpolatedAssetPropertyValues.html#API_GetInterpolatedAssetPropertyValues_RequestSyntax

maybe better to use a string?  in the UI and check we can still use the boolean toggle, but this will be nicer when they add smooth :)